en/content_how_to_get_compensated_for_work.md

```diff
@@ -1,47 +1,59 @@
+## Overview
 
-## What Kind of Work is Available at SCRF?
+This document covers how to track hours, create an invoice, alternative payment methods, and important notes. Recently onboarded contributors and recipients of grants, awards, or rewards need to follow the steps below to receive compensation.
 
-There’s a wide variety of work at SCRF, from writing Research Summaries to copy editing and graphic design to various administrative roles. We support flexible disbursement schedules, honoraria, grants, and other arrangements.
+## Tracking Hours
 
-As researchers become more deeply engaged with projects, or take on larger pieces of work, we have longer-term relationships available, including salaried positions.
+Anyone working for SCRF is expected to track their hours using the template sheet and process outlined below. Anyone submitting bounties or non-hours-based grants can ignore this step.
 
-_We pay in cryptocurrency. For longer commitments, alternative arrangements can be made._
+* Make a copy of the [Hours Template Sheet](https://docs.google.com/spreadsheets/d/19CmaqELKS_S2gWQIs2kddA91rv5zca-51mk_sOJ0gYs/edit#gid=0) and title it ‘[Your Name] Hours’.
+* Track time worked in 30-minute increments on the spreadsheet copy.
+* Provide a detailed description of work completed for each line item.
+* Submit the template at the end of the month with an invoice.
 
-If you cannot accept funds due to organizational or legal reasons, we can discuss alternative options. This could include:
+## Invoicing and Payment System
 
-* funding a project
-* conference travel
-* charitable donations to an endowment in lieu
-* credits for compute resources, archive nodes, etc
-* supporting a blockchain-related organization or event of your choice
+SCRF officially pays invoices once per month at the end of the month. SCRF may process another round of payments mid-month, in certain situations. **All invoices are due two full business days before the end of the month**. Any billable hours forgotten or worked after the invoice was submitted can be added to the next invoice with a description. SCRF uses Request Network for all payments.
 
-Pay varies according to the complexity of the assignment and the time involved. Contributors who commit to longer-term engagements and have produced a good body of work for the forum are encouraged to seek additional and more lucrative responsibilities.
+* Use an Ethereum address. (SCRF recommends [Metamask](https://metamask.io/) but any exchange wallet is acceptable when transferring funds to a fiat wallet right away.)
+* Create an account on [Request Network](https://app.request.finance/login).
+* Choose USD for invoice currency.
+* Select DAI stablecoin as the payment cryptocurrency.
+* Add a new client using the email address invoice@scrf.io.
+  * Smart Contract Research Forum
+  * 1250 Broadway
+  * 36th Fl
+  * 10001 New York
```

Comment:
It's a bit weird to have an address as list items. a code or a quote block would make more sense. 

```
 Smart Contract Research Forum
 1250 Broadway
 36th Fl
 10001 New York
```
or

> Smart Contract Research Forum
> 1250 Broadway
> 36th Fl
> 10001 New York